Cargo.toml

```diff
@@ -21,22 +21,25 @@ log = ["tracing/log"]
 default = ["tracing"]
 
 [dependencies]
-tower-service = "=0.3.0-alpha.2"
-tower-load = "=0.3.0-alpha.2"
-futures-util-preview = "=0.3.0-alpha.19"
-futures-core-preview = "=0.3.0-alpha.19"
-futures-sink-preview = "=0.3.0-alpha.19"
-tokio-executor = "=0.2.0-alpha.6"
-tokio-sync = "=0.2.0-alpha.6"
+tower-service = "0.3"
+tower-load = "0.3"
+futures-util = { version = "0.3", features = [ "sink" ] }
+futures-core = "0.3"
+futures-sink = "0.3"
+tokio = { version = "0.2", features = [ "sync" ] }
 crossbeam = "0.7"
 tracing = { version = "0.1.2", optional = true }
 pin-project = "0.4.0"
 
 [dev-dependencies]
-tokio = "=0.2.0-alpha.6"
+tokio = { version = "0.2", features = [ "full" ] }
```

Comment:
do you really wanna bring in full here?

Reply:
Well, this is for testing, so I'm not sure why not? We have several `#[tokio::test]` tests. I guess we could limit to just the single-threaded executor, but is there value in that?